duckduckgo/iOS
# iOS 13 SDK Support

Task/Issue URL: 
- https://app.asana.com/0/856498667320406/1128551253387167
- https://app.asana.com/0/856498667320406/1140757945084788
- https://app.asana.com/0/856498667320406/1140857123487538

**Description**:
Support for iOS 13 and Xcode 11, fixes to cookie handling and long press action sheet.

**Steps to test this PR**:

iOS 13 & UI
- Status bar style should contrast with screens like tabs list, home row instructions, settings etc.
- No more glitches on Privacy dashboard in dark/system dark themes.
- Edit bookmarks cell accessory color should always be visible (issue present when system light style was combined with app dark theme).
- Our Widgets should look good for both system themes.
- Cells padding for bookmarks.
- Actions sheets/alerts should be matching any selected in-app theme.
- Long press should work in Web View, query string parameters should be stripped for better clarity.

DDG Cookies preservation
- Set cookies through DDG settings (e.g. theme, languate etc).
- After fire button is pressed, these should be restored on first navigation.
- Check also new tab from empty state and app restoration after restart scenarios.

Supporting System theme
- Check settings screen functionality on both iOS 12 and 13.
- App UI should automatically match system theme and update if needed.
- Floating tip that points to the settings should be generic.

Onboarding
- Theme should not be covered during onboarding anymore. 

Pixels
- Entering Theme settings screen.
- Changing theme and leaving settings screen (also by swiping down).
- Changing theme, but then reverting the change, then leaving the screen.

Tests
- Unit & ATB tests should pass.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
